### DISCUSSION TITLE
Help needed in Prisma Migrate Upgrade (2.10.1 -> 2.30.0)

### DISCUSSION THREAD

**Discussion by rujhanArora** [prisma/prisma#11870] [Q&A] [answered]
We use Node.js and are currently upgrading Prisma as mentioned in the title. We have baselined in dev environment, deleted old migrations folder and created a new baseline migration file and it is working fine in our environment. But  We ship our product as a part of K8S cluster to our customers and thus have no direct control over the db. Thus, We are facing a couple of problems -
1. For old customers, we want to mark the new migration file as resolved, so that it doesn't mess up their current db. But for new customers, we want to run the new migration file to setup their db. How to achieve this?
2. If we add the resolve command as a part of application startup script for old customers, it will work for the first time, but for the second time, it will throw error that this migration has already been resolved.
Can the prisma team please help us here? Would be really helpful!

**Comment by Jolg42** [comment, marked as answer]
So in 1 it would be 
- `prisma migrate resolve --applied 20210426141759_initial-migration`

I feel like I'm missing some information to help you here:
Do you have one "codebase" deployed in a K8S cluster and each instance is for a different customer?
Do you have one database per customer?
When is the `prisma migrate resolve` called? Once for each instance during startup?

>thus have no direct control over the db

So you cannot access the database outside the cluster?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
